main/src/library/Graph.flix

```diff
@@ -108,6 +139,49 @@ namespace Graph {
     /// Returns the shortest distance between all pairs of vertices in the
     /// weighted directed graph `g`.
     ///
+    pub def stableDistances(g: m[(t, Int32, t)]): Option[Map[(t, t), Int32]] with Foldable[m], Boxable[t] = {
```

Comment:
Where does the name `stableDistances` come from?

Reply:
Oh yeah that's proabably not the best name. It just doesn't loop forever on negative cycles. You can compare it to the implementation in the examples folder